### ISSUE TITLE
[BUG] If prompt has a newline then everything after the new line is no longer shown in the image browser

### ISSUE THREAD

**Issue by vikas-movva** [cocktailpeanut/breadboard#23]
## here are some images to demonstrate the problem:

image 1:
![image](https://user-images.githubusercontent.com/65259115/211054990-702bbce9-6a81-45c5-91c7-761bfc0bdc33.png)

XML:
```<?xpacket begin="?" id="W5M0MpCehiHzreSzNTczkc9d"?>
<x:xmpmeta xmlns:x="adobe:ns:meta/" x:xmptk="Adobe XMP Core  5.6-c138 79.159824, 2016/09/14-01:09:01        ">
  <rdf:RDF xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#">
    <rdf:Description rdf:about="XMP template with common namespaces" xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:xmp="http://ns.adobe.com/xap/1.0/">
      <xmp:gm rdf:parseType="Resource">
        <xmp:prompt>
          masterpiece, best quality, high quality, highres,
        </xmp:prompt>
        <xmp:sampler>
          DPM++ SDE
        </xmp:sampler>
        <xmp:steps>
          20
        </xmp:steps>
        <xmp:cfg_scale>
          8
        </xmp:cfg_scale>
        <xmp:seed>
          2741978627
        </xmp:seed>
        <xmp:agent>
          automatic1111
        </xmp:agent>
      </xmp:gm>
    </rdf:Description>
  </rdf:RDF>
</x:xmpmeta>
<?xpacket end="w"?>
```

Same image in Auto1111 image browser:
![image](https://user-images.githubusercontent.com/65259115/211056489-f05b8f97-7446-4ada-b4e3-3205cf04ddec.png)

Other examples:
![image](https://user-images.githubusercontent.com/65259115/211056645-51eaf584-780b-45e4-98ee-911bd0dd00bf.png)

**Comment by cocktailpeanut** [comment]
@vikas-movva could you:

1. Zip some of those images
2. And share here?

I would like to debug using actual samples

**Comment by vikas-movva** [comment]
@cocktailpeanut Here is the zip file:
[example_images.zip](https://github.com/cocktailpeanut/breadboard/files/10365921/example_images.zip)

let me know if you need anything else.

**Comment by cocktailpeanut** [comment]
@vikas-movva This should be fixed with the v0.2.0 release which just dropped today :)

Check it out here: https://www.reddit.com/r/StableDiffusion/comments/10a3ryo/releasing_breadboard_020_codename_filterverse_a/

I tested with the attached image and have confirmed it works, and closing this issue for now, but please feel free to re-open if it didn't fix for you.

Just to clarify, in case it doesn't work at first for some reason, try re-indexing (can be found in the settings tab)
